src/main/java/com/denizenscript/denizencore/objects/core/ElementTag.java

```diff
@@ -814,6 +835,11 @@ public static void registerTags() {
         // Returns the element, with any contained tags parsed.
         // WARNING: THIS TAG IS DANGEROUS TO USE, DO NOT USE IT UNLESS
         // YOU KNOW WHAT YOU ARE DOING. USE AT YOUR OWN RISK.
+        // @example
+        // # This takes an input from chat and sets the player's health to it, and parses any tags in it.

```

Comment:
`chat` -> `a command argument`, `[input]` -> `<context.args.first>` or something

A command with tag input *might* make sense, chat almost definitely does not